lib/src/pubspec_config/pubspec_config.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:yaml/yaml.dart';
+
+part 'pubspec_config.freezed.dart';
+
+@freezed
+abstract class PubspecConfig implements _$PubspecConfig {
+  factory PubspecConfig({
+    String projectId,
+    String apiToken,
+    Iterable<String> includeTags,
+    String output,
+  }) = _PubspecConfig;
+
+  factory PubspecConfig.fromPubspecYamlString(String yamlString) {
+    final yaml = loadYamlNode(yamlString);
+    final flutterLokalise = yaml.getOrNull<YamlMap>("flutter_lokalise");
+    return PubspecConfig(
```

Comment:
Maybe you can return nil when flutter_lokalise doesn't exist in pubspec?

Reply:
I left it as it was - it doesn't provide much info right now.